cms/check_used_space

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+# Copyright 2012-2020 CERN
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Donata Mielaikaite, <donata.mielaikaite@cern.ch>, 2020
+# - Fernando Garzon, <oscar.fernando.garzon.miguez@cern.ch>, 2020
+# - Eric Vaandering, <ewv@fnal.gov>, 2021
+# - Panos Paparrigopoulos, <panos.paparrigopoulos@cern.ch>, 2024
+
+"""
+Probe to check used space.
+"""
+
+import sys
+import traceback
+
+from rucio.core.rse import list_rses, get_rse_usage, list_rse_attributes, get_rse_limits
+from rucio.db.sqla import models
+from rucio.db.sqla.session import get_session
+
+from utils import common
+
+PrometheusPusher = common.PrometheusPusher
+
+# Exit statuses
+OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
+
+if __name__ == '__main__':
+    try:
+        session = get_session()
+        with PrometheusPusher() as manager:
+            for rse in list_rses():
+                sources = get_rse_usage(rse['id'])
+                attributes = list_rse_attributes(rse['id'])
+                country = attributes.get('country', 'UNKNOWN')
+                rse_type = session.query(models.RSE.rse_type).filter(models.RSE.id == rse['id']).scalar()
+                rse_type = str(rse_type).split('.', 1)[1]
+                limits = get_rse_limits(rse['id'])
+                rucio_used = None
+                static_used = None
+                free_space = None
+                prom_labels = {'rse': rse['rse'], 'country': country, 'rse_type': rse_type, 'source': ''}
+                for usage in sources:
+                    
+                    # Calculate free rucio space of RSE and push it
+                    if usage['source'] == 'rucio':
+                        prom_labels['source'] = 'rucio_free_space'
+                        rucio_used = usage['used']
+                        rucio_free_space = int(usage['total']) - int(usage['used'])
+                        (manager.gauge(name='report_used_space.{rse}.{country}.{rse_type}.rucio_free_space',
+                                   documentation='Space used at an RSE from various sources')
+                         .labels(**prom_labels)
```

Comment:
I'd like to take a look at this metric name and see if it makes sense. Maybe we want to break it, call the metric "rucio_free_space" (and "free_space" below, etc). This is the one set of metrics that I know which have labels in the middle, in a sense. I don't think report used space tells us anything. First, it's no longer the name of the probe and it's in the job label anyhow.